pom.xml

```diff
@@ -13,39 +13,22 @@
  * information: "Portions copyright [year] [name of copyright owner]".
  *
  * Copyright 2017 ForgeRock AS.
- * 
+ *
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+    <parent>
+        <artifactId>openam-auth-trees</artifactId>
+        <groupId>org.forgerock.am</groupId>
+        <version>6.5.0</version>
+    </parent>
```

Comment:
Shouldn't inherit from AM codebase. Here's a recently updated POM that would be a good example of how this POM should look like:
https://github.com/ForgeRock/set-profile-property-auth-tree-node/blob/master/pom.xml

Reply:
used new POM